packages/facade/src/apis/sheets/f-worksheet.ts

```diff
@@ -947,4 +948,28 @@ export class FWorksheet {
      * @param range The range to set as the active selection.
      */
     setActiveSelection = this.setActiveRange;
+
+    /**
+     * Sets the frozen state of the current sheet.
+     * @param {IFreeze} freeze - The freeze object containing the parameters for freezing the sheet.
+     * @returns {boolean} True if the command was successful, false otherwise.
+     */
+    setFreeze(freeze: IFreeze): boolean {
```

Comment:
Can the following API be implemented?

- [setFrozenColumns(columns)](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#setFrozenColumns(Integer))	void	Freezes the given number of columns.
- [setFrozenRows(rows)](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#setFrozenRows(Integer))	void	Freezes the given number of rows.
- [getFrozenColumns()](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#getFrozenColumns())	Integer	Returns the number of frozen columns.
- [getFrozenRows()](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#getFrozenRows())	Integer	Returns the number of frozen rows.

Reply:
and hope you can add some test code

Reply:
add getFreeze